docs/user/rstudio/ide/guide/tools/jobs.qmd

```diff
@@ -2,98 +2,107 @@
+
+-   **Copy job results**: By default, the temporary workspace in which the job runs is not saved. If you'd like to import data from your job back into your R session, you have a couple of choices:
+
+  -   **Global environment**: This places all the R objects your job creates back in your R session's global environment. 
+  
+::: {.callout-caution}
 
-**Copy job results**: By default, the temporary workspace in which the job runs is not saved. If you'd like to import data from your job back into your R session, you have a couple of choices:
+Use **Copy jobs results** to **Global environment** option with caution! The objects created by the job will overwrite, without a warning, any objects that have the same name in your environment.
 
-*Global environment*: This places all the R objects your job creates back in your R session's global environment. Use this option with caution! The objects created by the job will overwrite, without a warning, any objects that have the same name in your environment.
+:::
 
-*Results object*: This places all the R objects your job creates into a new environment named `yourscript_results`.
+  -   **Results object**: This places all the R objects your job creates into a new environment named `yourscript_results`.
+
+An additional option is to include specific R code to save out the results to files on disk such as `readr::save_rds("results.rds")`
 
 ### Lifetime
 
-Background jobs run as non-interactive child R processes of your main R process, which means that they will be shut down if R is. While your R session is running jobs:
+Background jobs run as non-interactive child R processes of your main R process, which means that they will be shut down if R is. While the R session is running jobs:
 
 -   You will be warned if you attempt to close the window while jobs are still running (on RStudio Desktop)
 
 -   Your R session will not be suspended (on Posit Workbench)
 
 While background jobs are running, a progress bar will appear in the R console summarizing the progress of all running jobs.
 
-![](images/background-job-console.png "R console pane showing job progress tab"){alt="" width="350"}
+![](images/background-job-console.png){fig-alt="R console pane showing job progress tab" width="350"}
 
 ### Detailed progress
 
 The progress bar RStudio shows for your job represents the execution of each top-level statement in your R script. If you want a little more insight into which part of the script is currently running, you can use RStudio's [code sections](https://support.rstudio.com/hc/en-us/articles/200484568-Code-Folding-and-Sections) feature. Add a section marker like this to your R script:
```

Comment:
Change code sections link to [updated page](https://colorado.rstudio.com/rsc/rstudio-user-guide-2022-09-29/ide/guide/code/code-sections.html)

Reply:
Changed - thanks!